### DISCUSSION TITLE
Porta smtp

### DISCUSSION THREAD

**Discussion by manuelbarberi** [iisgiua/giuaschool#200] [Richieste di aiuto]
Come posso configurare mail... non funziona

**Comment by trinko** [comment]
Usa la modalità GMAIL, che solitamente funziona sui vari hosting. 
Per le altre modalità devi avere i parametri di configurazione che l'hosting ti fornisce.

**Reply by manuelbarberi** [reply]
A me non funziona nemmeno configurando bene

**Comment by trinko** [comment]
Scarica la versione 1.4.3, esegui la procedura di aggiornamento e configura l'email.
